src/components/Contact/Header.js

```diff
@@ -30,28 +24,10 @@ const styles = StyleSheet.create({
 
   header: {
     color: 'white',
-    fontSize: 25,
+    fontSize: 35,
```

Comment:
Contact/Header.js: Font size of header has been changed to a larger size to differentiate from the text in the content. Code is clean and well-organized for future additions/amendments.